automation/taskcluster/decision_task_pull_request.py

```diff
@@ -0,0 +1,164 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import datetime
+import json
+import os
+import taskcluster
+import re
+import subprocess
+import sys
+
+
+"""
+Decision task for pull requests
+"""
+
+TASK_ID = os.environ.get('TASK_ID')
+REPO_URL = os.environ.get('GITHUB_HEAD_REPO_URL')
+BRANCH = os.environ.get('GITHUB_HEAD_BRANCH')
+COMMIT = os.environ.get('GITHUB_HEAD_SHA')
+
+def fetch_module_names():
```

Comment:
Should we remove this and the following functions? It makes the file harder to read.